modules/serverless-openshift-routes.adoc

```diff
@@ -1,7 +1,12 @@
 [id="serverless-openshift-routes_{context}"]
 = Configuring {product-title} routes for Knative services
 
-If you want to configure a Knative service to use your TLS certificate on {product-title}, you must disable the automatic creation of a route for the service by the {ServerlessOperatorName}, and instead manually create a `Route` resource for the service.
+If you want to configure a Knative service to use your TLS certificate on {product-title}, you must disable the automatic creation of a route for the service by the {ServerlessOperatorName}, and instead manually create a route for the service by using the following procedure.
```

Comment:
```suggestion
If you want to configure a Knative service to use your TLS certificate on {product-title}, you must disable the automatic creation of a route for the service by the {ServerlessOperatorName} and instead manually create a route for the service.
```

> Avoid self-referential text whenever possible, such as "This topic is about" or "This section describes". Instead, focus on the purpose or value of the content, and include words that users are likely to search for. 

I think that if you're going to explicitly refer to the steps that the reference in the note makes more sense.